runtime/error.c

```diff
@@ -181,3 +187,83 @@ TaggedPtr BAL_LANG_ERROR_NAME(message)(TaggedPtr error) {
     ErrorPtr ep = taggedToPtr(error);
     return ep->message;
 }
+
+void _bal_print_mangled_name(const char *mangledName, FILE *fp) {
+    // If mangled name starts from _B_, the identifier is non-public
+    if (mangledName[2] == '_') {
```

Comment:
This code makes me sad. First, you must not assume that the first two chars are `_B`. More importantly, you must not assume that mangledName has length > 1.

Reply:
Fixed